tests/plugins/fail_by_itself.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+from pyln.client import Plugin
+import os
+import threading
+import time
+
+plugin = Plugin()
+
+
+class FailThread(threading.Thread):
+    def __init__(self):
+        super().__init__()
+        self.start()
+
+    def run(self):
+        time.sleep(1)
+        print("Exiting!")
+        os._exit(1)
```

Comment:
Interesting, I usually use `sys.exit(1)` for these, but the docs explicitly suggest using `os._exit(1)` after a `fork` call.